do-k8s/go.mod

```diff
@@ -0,0 +1,8 @@
+module demo-site
+
+go 1.14
```

Comment:
go 1.14 ? :o seems a bit low don't you think ?

Reply:
Yea, I think this was generated.